Sources/PostgresNIO/New/PSQLEventsHandler.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 19.01.21.
+//
+
+import NIOTLS
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIOTLS
```